src/registrar/templates/application_org_election.html

```diff
@@ -5,11 +5,16 @@
   <h2 class="margin-bottom-05">
     Is your organization an election office?
   </h2>
+
+  <p class="text-semibold">This question is required.</p>
 {% endblock %}
 
+{% block form_required_fields_help_text %}
+{# commented out so it does not appear on this page #}
+{% endblock %}
```

Comment:
My only overall suggestion would be to consider explicitly placing the asterisk on the visible "label":

```
Is your organization an election office?
  <abbr class="usa-hint usa-hint--required" title="required">*</abbr>
```

and displaying a consistent help text throughout the form (at least on the pages which have required questions). I think self-similar wording combined with a splash of red color will reduce cognitive load and help users with cognitive impairments. There's some [supplemental guidance](https://www.w3.org/WAI/WCAG2/supplemental/patterns/o1p03-consistent-design/) from WCAG, although I might be applying it in an overly broad manner.

I'll defer to @igorkorenfeld on this. Igor, could you consider the matter and leave a reply for Neil?

Reply:
Thanks for tagging me in on this Seamus. 

Adding the asterisk to the question sounds good to me, if keep the "This question is required" text below it, closer to the input field (I wanted the proximity of the note to the input field). The only trouble is that I wouldn't want to have a legend for the asterisk in addition to the "This question is required."